libdnf/module/module_item.cpp

```diff
@@ -101,6 +110,16 @@ std::string ModuleItem::get_name_stream_version() const {
 }
 
 
+std::string ModuleItem::get_name_stream_context() const {
+    return libdnf::utils::sformat(
+        "{}:{}:{}",
```

Comment:
I am not sure whether it is the fastest way how to merge multiple strings. Please can you add TODO to play here more with performance?